cvmfs/cvmfs.cc

```diff
@@ -178,6 +178,18 @@ class FuseInterruptCue : public InterruptCue {
   fuse_req_t *req_ptr_;
 };
 
+/**
+ * Options related to the fuse kernel connection. The capabilities are
+ * determined only once at mount time. If the capability trigger certain
+ * behavior of the cvmfs fuse module, it needs to be re-triggered on reload.
+ * Used in SaveState and RestoreState to store the details of symlink caching.
+ */
+struct FuseState {
+  FuseState() : version(0), has_dentry_expire(false) {}
+  unsigned version;
+  bool has_dentry_expire;
```

Comment:
i think we also need to save `cache_symlinks_` . it is set at the same location in `cvmfs.cc` as the `fuse_expire_entry_` and it also depends on if the capability is available

(the default is `cache_symlinks_ i = false`, if `CVMFS_CACHE_SYMLINKS` is set it is true, but only in `cvmfs.cc` it will double-check and disable if the capapbility is not supported)

Reply:
Yep, thanks! That has already be [done|https://github.com/cvmfs/cvmfs/pull/3347/commits/92f4fa0c66c46499d55b343ea028e7066f204efe]

Reply:
my bad.. should have f5